tests/robot/CONDITION/02_search.robot

```diff
@@ -33,7 +33,7 @@ Force Tags              search
 001 Search Diagnose Condition
     [Documentation]     Search Diagnose Condition
 
-    condition.create diagnose condition    create.json
+    condition.create diagnose condition    create-default.json
```

Comment:
I will refactor this in your other branch
https://github.com/ehrbase/fhir-bridge-poc/tree/feature/NUM-544_Repro_refactoring-Condition